website/versioned_docs/version-0.13.1/flink-quick-start-guide.md

```diff
@@ -98,7 +98,7 @@ dependency to your project:
 <dependency>
     <groupId>org.apache.hudi</groupId>
     <artifactId>hudi-flink1.13-bundle</artifactId>
-    <version>0.13.1/version>
+    <version>0.13.1/<version>
```

Comment:
```suggestion
    <version>0.13.1<version>
```

Reply:
@danny0405 thanks for the review, since the origin didn't fixed the typo and I create another PR for the fixes, do you mind take a look? https://github.com/apache/hudi/pull/9792